.sanity-ansible-ignore-2.10.txt

```diff
@@ -1,3 +1,4 @@
-plugins/modules/nbde_server_tang.py pylint:blacklisted-name
-plugins/modules/nbde_server_tang.py pylint:ansible-format-automatic-specification
-plugins/modules/nbde_server_tang.py validate-modules!skip
+plugins/modules/nbde_server_tang.py validate-modules:doc-choices-do-not-match-spec
+plugins/modules/nbde_server_tang.py validate-modules:missing-gplv3-license
+plugins/modules/nbde_server_tang.py validate-modules:parameter-type-not-in-doc
+plugins/modules/nbde_server_tang.py validate-modules:undocumented-parameter
```

Comment:
`doc-choices-do-not-match-spec`, `parameter-type-not-in-doc`, `undocumented-parameter` are usually straightforward to fix - were you planning on a separate commit for those?

Reply:
> doc-choices-do-not-match-spec, parameter-type-not-in-doc, undocumented-parameter are usually straightforward to fix - were you planning on a separate commit for those?

I was worried the fixes might be "invasive", but if they are straightforward, I'm going to look into them some more. Thanks.

Reply:
I think they are just doc fixes - make the docs embedded in the python file accurately describe the parameters in the python code.